resources/views/hr/application/index.blade.php

```diff
@@ -46,167 +45,153 @@
         $search = request()->has('search') ? '&search=' . request('search') : '';
         $query_filters = $hr_job_id . $search
     @endphp
-    <div class="d-flex align-items-center justify-content-between">
-        <ul class="nav nav-pills mb-2">
-            <li class="nav-item">
-                <a class="nav-item nav-link d-flex align-items-center {{ $status ? 'text-info' : 'active bg-info text-white' }}"
-                    href=/{{ Request::path() }}?status={{ config('constants.hr.status.new.label') }}{{$query_filters}}> <i
-                    class="fa fa-clipboard"></i>&nbsp;
-                    Open
-                    <span
-                        class="ml-1 d-inline-block px-2 py-0 {{ $status ? 'bg-info text-white' : 'active bg-white text-info' }}"
-                        style="border-radius: 20px;font-size: 12px;font-weight: 700;">
+    <div class="menu_wrapper">
+        <div class ="navbar"  id="navbar">
+            <li id="list-styling">
+                <a id="job-application-listings" class= "{{ $status === config('constants.hr.status.new.label') ? 'job-application-status' : '' }} btn"
+                    href=/{{ Request::path() }}?status={{ config('constants.hr.status.new.label') }}{{$query_filters}} >
+                    <sup class = "application-menu-options-title fz-20">
                         {{$newApplicationsCount + $inProgressApplicationsCount - $trialProgramCount}}
-                    </span>
+                    </sup>
+                    <svg class="job-application-icons">{!! file_get_contents(public_path('icons/people.svg')) !!}</svg>
```

Comment:
You cannot use svg tag outside an svg. Use span tag like this and make the design changes as the UI is getting changed by this.
```suggestion
                    <span class="d-block h-24 w-24 job-application-icons">{!! file_get_contents(public_path('icons/people.svg')) !!}</span>
```

Reply:
@GauravGusain98 this is happening after using span class.
![Screenshot (108)](https://user-images.githubusercontent.com/79009032/134100916-a24f17d5-6e14-471b-bbb9-638f1ceb794e.png)

Reply:
This is sorted but, how to align the icons to the center. I tried using `align-items: center;` using span class but changes are not visible.
![Screenshot (109)](https://user-images.githubusercontent.com/79009032/134115999-637edb62-44b8-4c4e-b81e-83416e632d14.png)